guides/user/applications/create.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: single
+title:  "Create an application"
+sidebar:
+  nav: guides
+---
+
+You can't create a deployment pipeline unless you have an application to put it
+in. If you're accessing Spinnaker for the first time and there are no
+applications available, the first thing you're going to do is create one.
+
+1. From any screen, click on the **Applications** tab.
+
+   Note that there might be some (unconfigured) applications there already.
+   these might be created automatically, based on existing infrastructure, but
+   they're not suitable for putting your pipelines in; go ahead and create a
+   new one.
+
+1. Click **Actions**, then **Create Application**.
+
+   ![](/guides/user/applications/create_application.png)
+
+1. Provide the application attributes in the **New Application** dialog.
+
+   * Resources created inside this application will follow the
+   [application-stack-detail naming convention](/concepts/#the-application-stack-detail-naming-convention/),
+   such that **Name** is used for application.
```

Comment:
Is it worth noting that the name can't be changed once the application is created?

Reply:
Couldn't hurt.